src/FFmpeg.cpp

```diff
@@ -518,21 +178,38 @@ class FindFFmpegDialog final : public wxDialogWrapper
 
    void OnBrowse(wxCommandEvent & WXUNUSED(event))
    {
+      static const FileNames::FileTypes types = {
+#   if defined(__WXMSW__)
+         { XO("Only avformat.dll"), { wxT("avformat-*.dll") } },
+#   elif defined(__WXMAC__)
+         { XO("Only avformat.dll"), { wxT("ffmpeg.*.64bit.dylib") } },
+#   else
+         { XO("Only avformat.dll"), { wxT("libavformat.so.*") } },
```

Comment:
Should that be `libavformat*.so.*`

Reply:
These string literals for file globs are repeated in this file.  Better to remove repetition.